app/feature/feature-terminate-insurance/src/main/kotlin/com/hedvig/android/feature/terminateinsurance/navigation/TerminateInsuranceDestination.kt

```diff
@@ -3,15 +3,13 @@ package com.hedvig.android.feature.terminateinsurance.navigation
 import com.hedvig.android.data.contract.ContractGroup
 import com.kiwi.navigationcompose.typed.Destination
 import kotlinx.datetime.LocalDate
+import kotlinx.serialization.SerialName
 import kotlinx.serialization.Serializable
 
 @Serializable
 data class TerminateInsuranceFeatureDestination(
-  val contractId: String,
-  val insuranceDisplayName: String,
-  val exposureName: String,
-  val contractGroup: ContractGroup,
-  val activeFrom: LocalDate,
+  @SerialName("insuranceId")
+  val contractId: String?,
```

Comment:
Now I know we use `insuranceId` for the route name, but why do we have this split here? Is there a world where these two could be named the same thing?
If it means that the route will also change to contractId the best time to do it is now before the deep link is out in production and it's not possible to change anymore without breaking old clients.
Otherwise, maybe the parameter should be named insuranceId as well? Not sure if those two imply any different meaning really.